### ISSUE TITLE
Custom HDTV quality settings grabbing SD/MP4 files

### ISSUE THREAD

**Issue by PhilJayJ** [SiCKRAGETV/sickrage-issues#1569]
SR Version:  BRANCH: (master) / COMMIT: (bf2baf309bd01683fdc06d57ece0b89d53b4a180)
Python Version: 2.7.6 (default, Mar 22 2014, 22:59:56) [GCC 4.8.2] 

Not sure if there is a bug in the code or I have the wrong settings but I've tried many things to try and fix my issue (IRC, various forums and SR re-install) but nothing works, so please bear with me - I'm extremely new to this.

I've added a number of shows to SR and set the quality to 'Custom'. From there, I've set the 'Initial' and 'Archive' quality to HDTV. My understanding from reading the Quality Settings FAQ page is that SR should only be grabbing 720p/HDTV/x264 files with the above settings. However, it's downloading MP4/HDTV/SD files.

Looking at the Quality Settings FAQ page - https://github.com/SiCKRAGETV/SickRage/wiki/Quality-Settings - the HDTV quality setting setting should only grab: 

File name contains 720p and HDTV and x264, and not 1080[i/p]
or:
File name contains "hr.ws.pdtv.x264", and not 1080[i/p]

Everything I've ever downloaded is 720p HDTV x264 and would like to keep it like this. But despite my best efforts I'm still getting MP4/HDTV/SD files.

Is there a bug in the code or do I have the wrong settings? I have tried a non-custom setting (HD720p) but I get WEB-DL files which I don't want.

Many thanks

**Comment by neoatomic** [comment]
Why set the quality to custom.? If you want HDTV than just set it to HDTV.?
If SD quality is downloaded when HDTV is set than its probably bad naming of the torrent.
Try to witch to another torrent provider.

Here is a good guide for configuring SR, that should get you a good start.
http://www.htpcguides.com/configure-sickrage-usenet-torrent-tv/

**Comment by PhilJayJ** [comment]
I set it to custom because the HDTV setting grabs WEB-DL stuff as well, but I only want 720p HDTV. I tried downloading one series which had an 'MP4/x264/HDTV/SD/Scene' and 'MKV/x264/HDTV/720p/Scene' file. Even though I set my quality pref to custom/HDTV, it grabbed the MP4. I used required words of x264, HDTV, 720P etc and it still grabbed blurays and 1080p files.

Thanks for the guide, I'll give it a go.
